src/main/java/net/sacredlabyrinth/Phaed/PreciousStones/managers/PermissionsManager.java

```diff
@@ -42,6 +44,7 @@
     private Lockette lockette = null;
     private PreciousStones plugin;
     private mcMMO mcmmo = null;
+	private WorldEditPlugin worldEdit = null;
 
```

Comment:
Seems like this isn't really needed though, fortunately. This could just be a boolean flag and then we don't need the class reference